brage-import/src/main/java/no/sikt/nva/brage/migration/model/Record.java

```diff
@@ -0,0 +1,247 @@
+package no.sikt.nva.brage.migration.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.net.URI;
+import java.nio.file.Path;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import no.sikt.nva.brage.migration.model.content.ResourceContent;
+import no.sikt.nva.brage.migration.model.entitydescription.EntityDescription;
+import no.sikt.nva.brage.migration.model.entitydescription.PublishedDate;
+import nva.commons.core.JacocoGenerated;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+
+@JacocoGenerated
+@SuppressWarnings("PMD.TooManyFields")
+public class Record {
+
+    private EntityDescription entityDescription;
+    private URI customerId;
+    private URI id;
+    private URI doi;
+    private Path origin;
+    private Type type;
+    private Language language;
```

Comment:
Why isn't language set in EntityDescription?

Reply:
Ok, this class is a bit weird, maybe I'm missing some detail in what it does

Reply:
It's an in-between datamodel. It is closely matched to NVA, but according to product-owner he wants to preserve the Brage-publications raw data. This because several fields have been converted from a "brage-type" to a NVA-type. (ex. brage publication instance type is an array which will be combined to a single NVA publication-instance type: https://github.com/BIBSYSDEV/nva-brage-migration/blob/main/src/main/java/no/sikt/nva/scrapers/TypeMapper.java